test/instrumentation/decode_metric.go

```diff
@@ -130,10 +130,11 @@ func decodeLabels(expr ast.Expr) ([]string, error) {
 		if !ok {
 			return nil, newDecodeErrorf(bl, errLabels)
 		}
-		if bl.Kind != token.STRING {
+		value, err := stringValue(bl)
+		if err != nil {
 			return nil, newDecodeErrorf(bl, errLabels)
```

Comment:
Is this error supposed to be `errLabels`? I would think you would actually want to use the error returned from `stringValue`.